### PR TITLE
Makes msay be enabled by default 💬

### PR DESCRIPTION
**Pull request by FlattestGuitar**
This follows a staff discussion on the matter.

Not visible to players, so no CL.

👌

### CODE REVIEW

**Comment by Necaladun** [comment]
I thought this would be a problem and was against it to begin with.

I've changed my mind since.

**Comment by Aurorablade** [comment]
No it is a problem. No

**Comment by Anoonki** [comment]
I thought it was agreed upon that this wouldn't be a thing.
I'm certainly not against it, because the reason (if I remember rightly) was "Muh metagaming" but "Msay" of all the things is probably the least likely to ever be used for that anyways.

**Comment by Birdtalon** [comment]
I don't think it's a problem. It would be literally the most obvious thing I can think of for admins to see metagaming as it's plastered right into their text boxes. Can't be abused unless no admins online.

**Comment by sothangel** [comment]
Might be worth posting the results, if it was a unanimous/large majority/slim margin for yes.

**Comment by Spacemanspark** [comment]
@Aurorablade what's wrong with it?
It gets toggled on every round for them anyways, and I've yet to see one instance of metagaming with it. It honestly helps the memetors with communication.

**Comment by Draconic-saint** [comment]
I've always wondered **why** mentors need msay.

All they do is individualy contribute by trying to answer questions to their best ability.
You don't know? No issue. Someone else might. We already have the "take ticket" system or whatever the hell it's called to avoid people getting either ignored or overwhelmed.

So why do they need a seperate, log filling chat? I honestly can't think of a thing they NEED to communicate to **eachother** about via msay. And they already have a direct line to us via ahelp.

It can't compare to asay in function, because of the fundamental differences between mentor and admin. Moderators had a job that also required more communication. They had msay for a valid reason.

In the end, msay, to me, is rather redundant. I only see the use if I need to ask all mentors a question as admin. It's easier and I don't have to end up in a bwoinkstorm because I had to ask over PM.

The metagaming arguements are void, everything get's so logged nowadays, you can't even forcesay *fart on @Spacemanspark without getting caught.

TL;DR: I don't think mentors need msay, I also don't want another source of logs and text enabled by default as an admin. I'm biased, bite me.

**Comment by SamHPurp** [comment]
From when I was a mentor, msay is really valuable for when mentors take questions, and either don't fully know the answer and other mentors can pitch in. A few times I've been able to correct another mentor, so that they could correct themselves to the player asking. This is far better than correcting the mentors answer TO the player... it makes the mentors seem more competent, which is only a good thing.

I don't think mentors NEED it... but there is no legitimate reason to deny them it.

**Comment by davidchan13** [comment]
If you're worried about mentors metagaming then you should select better, trustworthy people to be mentors.  Given that many of the questions tossed at mentors to begin with are antag related.

As far as msay goes, seem silly not to have it. If a question gets lodged at the mentors, they can collaborate or 'claim' a question to ensure a person is getting accurate information without being flooded with the same response 3-5 times over. 

And if a mentor isn't sure they can throw the answer they want to give in msay for factchecking and proofreading before passing the info down.

**Comment by Anticept** [comment]
Perhaps a middle ground for now. Msay enabled by default while an admin is online?

**Comment by Citinited** [comment]
Mentors already have to stonewall not knowing the roundtype (as we almost always get mhelps about antag problems). This means anyone who's been a mentor for a while has proven that they don't metagame on information given to them OOCly.

Msay is actually used quite a bit by mentors, so :+1: to having it turned on by default.

**Comment by tylerh95** [comment]
Assuming the people against it are that way for "muh metagaming", that doesn't seem like it would be an issue. 

1) People picked for mentor should be trusted not to metagame. As stated by someone else, we often get questions that involve antags or other current round events. 

2) I think it would be a pretty rare occasion that a mentor would metagame, however even rarer would be two mentors who have agreed to metagame together over msay. 

3) Even if two mentors did decide to metagame, I'd take a guess that the vast majority would be smart enough to not use an in-game and logged method of communication to do it. Especially because any other online mentors or admins would see it.

**Comment by scrubmcnoob** [comment]
To be honest, if a mentor is going metagame and use mentorchat for metagaming it will only make weeding out the bad noodles the ever more easier.
Let's be real here, if a person wants to metagame, regardless of mentor status, they can easily use Skype or discord or any third party program which makes it much harder for us to catch.

And the other concern that was sparingly brought up was that it would turn into a private ooc and clutter chat. It's not hard for an admin to tell mentors to chill on the talking in msay but even when we have it enabled on the server, this is not a problem.

**Comment by Fox-McCloud** [comment]
>If you're worried about mentors metagaming then you should select better, trustworthy people to be mentors.

The standards currently established for mentors being selected for mentorship are not at level high enough for this level of trust, in my opinion; we select primarily for game knowledge first, with a distant second placed on how "agreeable" the person is.

**Comment by SamHPurp** [comment]
@Fox-McCloud sounds like a cop out.

The only arguement against msay being enabled by default is that it'd be treated as an OOC chat for the 'cool kids' - which was the arguement I've heard presented many times by others such as @Regen1 . This is a legitimate arguement, but not one I've seen happen in the many many rounds that msay gets enabled at round start.

And if it does, its fairly easy to say 'knock it off' - or just strip mentor from them if they persist.

**Comment by sothangel** [comment]
Perhaps the administrative staff shouldn't be so freely handing out of mentor roles if, as Fox implies; not being a shit with the sensitive information you have is secondary to knowing the game, but that isn't the scope of this PR.

From my own experience, msay is useful to ask "Anyone know the answer to XYZ's question?", to correct and inform other mentors if they're unsure, and things like that. It's a minor thing really, but a nice QOL thing to have.

**Comment by ExitGame** [comment]
To be honest, from what I have seen, attitude/trust seems to come before game knowledge. Otherwise I would have been Mentor for a way longer time.

**Comment by SamHPurp** [comment]
Nine thumbs up and 1 thumbs down from Fox.

Here comes it being closed due to 'negative response'.

**Comment by saywatthe15th** [comment]
Helps mentors if they need to figure out a question and well, we trust the mentors not to misuse this.

**Comment by Anticept** [comment]
We can always disable it by default again if it truly turns out to be a problem.

**Comment by tylerh95** [comment]
@Fox-McCloud 

What level of trust? The level of trust needed to access a logged system of communication between people who are already trusted not to metagame based on the questions they get (that are often antag based)? 

As I tried to make clear in my previous comment, it seems like the only time this could really be used to metagame would be when no admins are on AND all of the online mentors are involved. It's not like all of the mentors are in cahoots against the server, and I'm sure the vast majority of the mentors would have no issue reporting a fellow mentor for metagaming in msay. 

Not allowing acccess to msay doesn't do anything at all to prevent mentors from metagaming via the endless methods of out of game communication that aren't logged.

**Comment by Draconic-saint** [comment]
@SamHPurp It's been established a lot of times that thumbs don't mean shit when it comes to maint final ruling... don't make/take it so personal either.

**Comment by tigercat2000** [comment]
@SamHPurp Upvotes/downvotes mean less than nothing on this. This is purely a policy decision and as such is fully up to the maintainers and headmins, however they decide to come to that conclusion.

**Comment by scrubmcnoob** [comment]
> The standards currently established for mentors being selected for mentorship are not at level high enough for this level of trust, in my opinion

So mentors can't be trusted with a simple command that allows them to talk in their own private chat in the game to make it more convenient for them but can be trusted with a private discord. 
Also, has the thought of if a mentor truly wanted to metagame, aren't there dem third party programs like myspace or something they could metagame over without us ever knowing? Just saying.

And isn't it a bit concerning we won't trust mentors with a simple chat command but trust them with plenty of other commands and being able to see player questions that can be easily used to figure the round type out. 
Something doesn't seem right here...

This should definitely be highlighted as the main point of the reason for it.
If a mentor really wanted to metagame, they can  just use a different program for communication and we literally have no way of telling unless their metagame is obvious.
If they start metagaming using msay, they'll just be banned and removed from the team.

Or do you find my statement of 'mentors can easily use other programs, that we have zero control over, to metagame' a false statement @Fox-McCloud

**Comment by theColdflame** [comment]
if mentors metagaming is even remotely a concern you need new mentors

**Comment by Anticept** [comment]
Again. If it's a problem, it's easy as hell to revert this. Or address it with msay rules. Or just pull that person's mentorship.

@Draconic-saint , forgot to respond to this. I've had cases where an mhelp progressed into something more complicated than the original question's scope. For that matter, since becoming a mentor, I've been bughunting *a lot* because players have had inconsistencies with either the wiki, or something broke recently in a patch. Know how I get around msay being disabled? I mhelp myself. This still generates the same amount of logs. Msay has been a godsend because I can ask other mentors if they know anything about it, and not continue to cause mass bwoinking unnecessarily.

**Comment by Dumbdumn5** [comment]
I am in full support of this.

When I was a mentor for a year, in albeit relatively different circumstances regarding rank, the job and role of mentor was very much the same as it is now. During that time, mentor chat was enabled at all times as brown chat that looked nothing like the logs and was well set apart from any of the other chats, and there wasn't an in-game toggle as far as I was aware, that would allow it to be turned off. Despite this lack of a quick toggle, the mentors often used the chat to discuss questions and game mechanics, throw out a few jokes, and get to know their team a bit more.

It might be nostalgia blindness, but I don't actually recall ever seeing mentor chat heavily abused to any point in which I would punish the entire rank thereafter due to the actions of one or a small group of mentors. Mentors are meant to be respectful, respectable, knowledgeable, and dependable, and to these lengths I don't believe the criteria has changed, just that the voting process has. 

Still, with this system, now well over a year old, there have continued to be competent and upstanding mentors selected through the process, and I don't personally see metagaming as an issue even if they didn't happen to be, as mentor chat is easily visible and can be filtered through in the logs with ease, if need be.

In addition, I believe that this system can act as a positive, not only for the reason that mentors can communicate the answer to a question more easily, but also for the sake of getting to know one another better. Though there is a discord channel for mentors, it's rarely used, much like the current channel for senior staff, it's used if something big happens or needs to be discussed, and is otherwise dead silent. An in-game chat like this might bring about some more discussion between admins and mentors, mentors and mentors, and might make things feel a bit more open and warm compared to being dropped in as they currently seem to be.

As said before, it's very easy to deal with if it becomes a problem, and the possibility for this to become one is very unlikely considering the criteria that should be and ,in my opinion, what remains to be in place in the selection of mentors.

**Comment by Anticept** [comment]
Apparently this was already discussed and hashed out in a staff thread. Please merge.

**Comment by Xantholne** [comment]
Honestly as it currently stands, just about every round with almost every different admin on Msay gets turned on by them because it lets us discuss certain things pretaining to questions. So it'd be nice to not have the hassle for admins to have to tourn it on for us.

**Comment by SamHPurp** [comment]
Support from every admin who has commented on this. Support from every mentor that has posted on this.

Fox disagrees, and therefore it won't get merged, or heels will be dragged in getting it merged. This is now a month on from the last comment. This is an administrative thing. 😄 🔫

**Comment by Fox-McCloud** [comment]
>Fox disagrees, and therefore it won't get merged, or heels will be dragged in getting it merged. This is now a month on from the last comment. This is an administrative thing. 

As @Necaladun already told you before; not everything is a conspiracy. This is an administrative matter, ergo, maintainers have little input on it; the "hold up" is because headmins, themselves, haven't come to a formal conclusion on the PR, so naturally it hasn't been merged yet.

So please, stop throwing around sarcastic accusations without any evidence to back them up.

**Comment by SamHPurp** [comment]
I do apologize, but saying something minorly inflammatory seems to get responses.

And by and large, that remains peoples biggest gripes right now. The lack of any form of communication on these PRs.

**Comment by Allfd** [comment]
@Xantholne its not a hassle.  We just assume one of the other admins did it already.
